tests/gis_tests/test_geoforms.py

```diff
@@ -346,6 +349,10 @@ class PointForm(forms.Form):
             self.assertNotIn(
                 "options['default_lat'] = %d;" % forms.OSMWidget.default_lat,
                 rendered)
+        if forms.OSMWidget.default_zoom != 17:
```

Comment:
To be honest I don't quite understand that condition, I simply copied it from the two examples above.